Wordcloud/livewordcloud.py

```diff
@@ -11,7 +11,7 @@
 import numpy as np
 import requests
 
-def gen_wordcloud(field: str = "title", data_folder='./') -> io.BytesIO:
+def gen_wordcloud(json_path: str, field: str = "title", data_folder='./') -> io.BytesIO:
```

Comment:
Please add new arguments at the end of the parameter list: otherwise any calls to `gen_wordcloud(name)` will stop working. I would also suggest that the old url is kept as the default argument, otherwise any calls already using the function will again be non-functional.

Please add it to the argument list for the function docstring as well (the string just below this line)

Reply:
Done.